websites/C/CodAre/dist/metadata.json

```diff
@@ -11,8 +11,8 @@
   },
   "service": "CodAre",
   "version": "1.0.0",
-  "logo": "https://barbecue.is-pretty.cool/88A9JqM.png",
-  "thumbnail": "https://barbecue.is-pretty.cool/5vWBXSv.png",
+  "logo": "https://imgur.com/f2U2Zd9",
+  "thumbnail": "https://imgur.com/9VV6XNf",
```

Comment:
use the i.imgur links

Reply:
> use the i.imgur links

im using hizliresim instead of imgur because imgur is so slow in turkey

Reply:
I want mention CONTRIBUTING.md has this line and you need to agree with it:
```All images in the metadata file must be hosted on i.imgur.com. Using content hosted on the website is not permitted as they can change the paths and files unwillingly.```